programs/witness_node/main.cpp

```diff
@@ -97,6 +97,19 @@ int main(int argc, char** argv) {
          return 1;
       }
 
+      cfg_options.add_options()
```

Comment:
Please deal with `help` and `version` before this.

Reply:
thank you, moved this.